src/scripts/main.js

```diff
@@ -1 +1,54 @@
 'use strict';
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    resolve('First promise was resolved');
+  });
+
+  setTimeout(() => {
+    reject(new Error('First promise was rejected'));
+  }, 3000);
+});
+
+firstPromise
+  .then((res) => createMessage(res, 'success'))
+  .catch((err) => createMessage(err, 'error'));
+
+const secondPromise = new Promise((resolve) => {
+  document.addEventListener('click', (e) => {
+    resolve('Second promise was resolved');
+  });
+
+  document.addEventListener('contextmenu', (e) => {
+    e.preventDefault();
+
+    resolve('Second promise was resolved');
+  });
```

Comment:
In the `secondPromise`, both the 'click' and 'contextmenu' events resolve the promise with the same message. This is correct according to the task requirements, but ensure that the promise is resolved only once. If both events are triggered, the promise should not resolve twice.